code/modules/clothing/under/miscellaneous.dm

```diff
@@ -707,3 +707,10 @@
 	item_state = "rottensuit"
 	inhand_states = list("left_hand" = 'icons/mob/in-hand/left/clothing.dmi', "right_hand" = 'icons/mob/in-hand/right/clothing.dmi')
 	_color = "rottensuit"
+
+/obj/item/clothing/under/medicalswimsuit
```

Comment:
There already seems to be an `/obj/item/clothing/under/swimsuit`, could you change this to that?